pypeit/spectrographs/keck_deimos.py

```diff
@@ -325,9 +342,14 @@ def config_specific_par(self, scifile, inp_par=None):
 
         headarr = self.get_headarr(scifile)
 
-        # When using LVM mask reduce only detectors 3,7
-        if 'LVMslit' in self.get_meta_value(headarr, 'decker'):
-            par['rdx']['detnum'] = [(3,7)]
+        # When using LVM mask or AMPMODE = SINGLE:A reduce only detectors 3,7
+        if ('LVMslit' in self.get_meta_value(headarr, 'decker') or
```

Comment:
Can you double check that this is `LVMslit` and not `LVMslitC`? I believe it should be the latter based on my raw frames.

Reply:
In the past, `LVMslitB` used to exists, so to take into account possible variation in name for the LVM mask, the condition here is just checking if `LVMslit` is part of the slitmask name.